docs/src/the-fuel-indexer.md

```diff
@@ -2,4 +2,6 @@
 
 ![Fuel Logo](./img/fuel.png)
 
-Fuel Indexer uses WASM to index transactions and state within a fuel network, allowing for high-performance read-only access to the blockchain for advanced dApp use-cases.
+The Fuel Indexer is a standalone binary that can be used to index various components of [the blockchain]. These indexable components include blocks, transactions, and [receipts](https://github.com/FuelLabs/fuel-specs/blob/master/specs/protocol/tx_format.md) and state within a fuel network, allowing for high-performance read-only access to the blockchain for advanced dApp use-cases.
```

Comment:
Looks like this might be missing a link - maybe needs to be copied from the README again?